core/server/worker/src/main/java/alluxio/worker/block/meta/StorageTier.java

```diff
@@ -82,6 +82,7 @@ private void initStorageTier()
     PropertyKey tierDirMediumConf =
         PropertyKey.Template.WORKER_TIERED_STORE_LEVEL_DIRS_MEDIUMTYPE.format(mTierOrdinal);
     String rawDirMedium = ServerConfiguration.get(tierDirMediumConf);
+    Preconditions.checkState(rawDirMedium.length() > 0, PreconditionMessage.ERR_TIER_MEDIUM_TYPE_BLANK);
```

Comment:
I would suggest not to use `PreconditionMessage`. It turned out a lot of issues in debugging